privacyidea/lib/resolvers/SQLIdResolver.py

```diff
@@ -728,15 +653,13 @@ def delete_user(self, uid):
         """
         res = True
         try:
-            conditions = []
-            conditions.append(self._get_userid_filter(uid))
+            conditions = [self._get_userid_filter(uid)]
             conditions = self._append_where_filter(conditions, self.TABLE,
                                                    self.where)
             filter_condition = and_(*conditions)
-            user_obj = self.session.query(self.TABLE).filter(
-                filter_condition).first()
-            self.session.delete(user_obj)
+            self.session.execute(self.TABLE.delete().filter(filter_condition))
```

Comment:
Is it `delete().filter()` or should it be `filter(...).delete()`.

or `...execute(delete(self.TABLE).filter(...)` ?

Reply:
You are right, according to the new style this should be something like
`self.session.execute(delete(self.TABLE).where(filter_condition))`
Will fix this

Reply:
Done in 54797b58